!!python/object/new:dynamic_reconfigure.encoding.Config
dictitems:
  ai_pilot_pitch: 0.0
  ai_pilot_roll: 0.0
  ai_pilot_x: 0.0
  ai_pilot_y: 0.0
  ai_pilot_yaw: 0.0
  ai_pilot_z: 0.0
  front_camera_pitch: 0.0
  front_camera_roll: 0.0
  front_camera_x: 0.0
  front_camera_y: 0.0
  front_camera_yaw: 0.0
  front_camera_z: 0.0
  front_velodyne_pitch: 0.0
  front_velodyne_roll: 0.0
  front_velodyne_x: 0.0
  front_velodyne_y: 0.0
  front_velodyne_yaw: 0.0
  front_velodyne_z: 0.0
  gps_pitch: 0.0
  gps_roll: 0.0
  gps_x: 0.0
  gps_y: 0.0
  gps_yaw: 0.0
  gps_z: 0.0
  groups: !!python/object/new:dynamic_reconfigure.encoding.Config
    dictitems:
      ai_pilot_pitch: 0.0
      ai_pilot_roll: 0.0
      ai_pilot_x: 0.0
      ai_pilot_y: 0.0
      ai_pilot_yaw: 0.0
      ai_pilot_z: 0.0
      front_camera_pitch: 0.0
      front_camera_roll: 0.0
      front_camera_x: 0.0
      front_camera_y: 0.0
      front_camera_yaw: 0.0
      front_camera_z: 0.0
      front_velodyne_pitch: 0.0
      front_velodyne_roll: 0.0
      front_velodyne_x: 0.0
      front_velodyne_y: 0.0
      front_velodyne_yaw: 0.0
      front_velodyne_z: 0.0
      gps_pitch: 0.0
      gps_roll: 0.0
      gps_x: 0.0
      gps_y: 0.0
      gps_yaw: 0.0
      gps_z: 0.0
      groups: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      id: 0
      imu_pitch: 0.0
      imu_roll: 0.0
      imu_x: 0.0
      imu_y: 0.0
      imu_yaw: 0.0
      imu_z: 0.0
      name: Default
      parameters: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      parent: 0
      state: true
      type: ''
    state: []
  imu_pitch: 0.0
  imu_roll: 0.0
  imu_x: 0.0
  imu_y: 0.0
  imu_yaw: 0.0
  imu_z: 0.0
state: []
